Added Steam Deck support, fixed Windows section, updated UE version support (#2304)
saucelabs/sauce-docs: docs/error-reporting/platform-integrations/unreal/setup.md

> #### For Crashes in Packaged Builds

Reviewer: ```suggestion
### For Crashes in Packaged Builds
```

> ##### Enable the Crash Reporter

Reviewer: ```suggestion
#### Enable the Crash Reporter
```

1. In the Unreal Editor, go to **Edit > Project Settings**.
1. In the Project Settings, search for "crash reporter".
1. Under **Packaging**, enable **Include Crash Reporter**.
   <img src={useBaseUrl('img/error-reporting/unreal-enable-crashreporter.png')} alt="Enable the Crash Reporter in the Unreal Editor." />

:::note
If you're building from the command line, add the `-crashreporter` flag.
:::

> ##### Configure the Crash Reporter

Reviewer: ```suggestion
#### Configure the Crash Reporter
```

To integrate error reporting in your Unreal Engine game for game consoles, see the [Console Integration Guides](https://support.backtrace.io/hc/en-us/sections/360007642051-Video-Game-Technologies).

</TabItem>
<TabItem value="SteamDeck">

> Follow the instructions for setting up crash reporting in Windows. The Steam Deck will not show the CrashReportClient after a crash so there will be no option for users to click the send button. To enable sending `-Unattendded` can be added to the launch options for the game. Please note this option sends crash reports without user intervention or knowledge like how the Windows client would normally allow.

Reviewer: ```suggestion
Follow the instructions for setting up crash reporting in Windows. The Steam Deck will not show the CrashReportClient after a crash, so there will be no option for users to click the send button. To enable sending `-Unattendded` can be added to the launch options for the game. This option sends crash reports without user intervention or knowledge like the Windows client normally allows.
```

To send a crash report to your Backtrace instance for Linux, see the [Crashpad Integration Guide](https://support.backtrace.io/hc/en-us/articles/360040516131-Crashpad-Integration-Guide#Sendcrashreports).

</TabItem>

<TabItem value="GameConsoles">

To send a crash report to your Backtrace instance for game consoles, see the [Console Integration Guides](https://support.backtrace.io/hc/en-us/sections/360007642051-Video-Game-Technologies).

</TabItem>
<TabItem value="SteamDeck">

> To crash your game when it starts, create a class called MyActor and reference a blueprint. The blueprint can be attached to the BeginPlay event.

Reviewer: ```suggestion
To crash your game when it starts, create a class called `MyActor` and reference a blueprint. The blueprint can be attached to the `BeginPlay` event.
```
